docs/design/y-flow/y-flow-persistence-model.puml

```diff
@@ -0,0 +1,50 @@
+@startuml
+
+title Y-Flow Persistence Model
+
+object YFlow <<Node>> {
+    y_flow_id
+    ..
+    status : FlowStatus
+    ..
+    shared_switch_id : SwitchId
+    shared_port
+    ..
+    maximum_bandwidth
+    path_computation_strategy : PathComputationStrategy
+    encapsulation_type : FlowEncapsulationType
+    max_latency
+    max_latency_tier2
+    ignore_bandwidth
+    periodic_pings
+    pinned
+    priority
+    strict_bandwidth
+    description
+    ..
+    y_point : SwitchId
+}
+
+object YSubFlowFields <<Relation>> {
+    y_flow_id
+    sub_flow_id
+    ..
+    shared_endpoint_vlan_id
+    shared_endpoint_inner_vlan_id
```

Comment:
what's the usecase for these?

Reply:
Discussed. Each sub-flow has own matching criteria for shared endpoint.